README.md

```diff
@@ -53,6 +53,87 @@ func main() {
+(e.g. internal employees), and different sets of data to other groups of
+clients (e.g. public internet). This has been largely exposed by allowing zones
+in the NS1 system to share the same FQDN, and allowing propagation to be
+controlled via ACLs and "views". For existing zones, and users that have no
+need for the added complexity of views, the default behavior is unchanged.
+However, it is important to understand that the requirement that an FQDN be
+unique within the network is removed in v3.x, and the ramifications of that.
+
+Views are available in API v3.x. For more information, see:
+https://help.ns1.com/hc/en-us/articles/360054071374
+
+COMPATIBILITY
+
+Since an FQDN can now appear in more than one "zone", it can no longer uniquely
+identify a zone. Instead, a user-supplied "name", unique within an account,
+is used to uniquely identify the zone.
+
+The zone name can be the same as its FQDN, existing zones transfer to the new
+schema this way. And if not using views, it does serve as a good identifier.
+However, if a second zone is created pointing to same FQDN, it cannot reuse
+the FQDN as an identifier, and queries for the zone FQDN (as an identfier)
+will return the first zone. The following example should help illustrate:
+
+In general, for API requests, the identifier for a zone will appear in the URL.
+They can also be passed or received as fields on an object. In the following
+request to a v2.x system, we use ZONE as an identifier in the URL, and may
+pass the `zone` field as an indicator of the FQDN for the zone.
+
+$ENDPOINT/v1/zones/example.com -d '{zone: example.com}'
+                   ^                ^
+                   L___ ZONE (id)   L____ ZONE (fqdn)
+
+Note also that it has been an ERROR if the values in the URL and `zone` field
+do not match, and the API would reject.
+
+Going forward, the unique identifier and FQDN of the zone are decoupled. The
+reference in the URL is user-assignable, and is passed and returned using the
+`name` field for zones (and the `zone_name` field for records). In the new
+paradigm, the previous call would look more like:
+
+$ENDPOINT/v1/zones/example.com -d '{zone: example.com, name: example.com}'
+                   ^                ^                  ^
+                   L___ NAME        L____ ZONE         L____ NAME
+
+For compatibility, if `name` isn't present, the API will use the FQDN, so the
+2.x call above should continue to work, and have the same result.
+
+However, in 3.x we are now allowed to make new zones with the same FQDN:
+
+$ENDPOINT/v1/zones/example-internal -d '{zone: example.com, name: example-internal}'
+                   ^                     ^                  ^
+                   L___ NAME             L____ ZONE         L____ NAME
+
+`example-internal` shares the FQDN with `example.com`. API calls using
+`example.com` as the identifier will uniquely identify the first zone, to
+address the second zone, `example-internal` must be used in the identifier
```

Comment:
It would read better to put a period or a semicolon after `the first zone`.

Reply:
done

Reply:
done